Gemfile.lock

```diff
@@ -0,0 +1,25 @@
+PATH
+  remote: .
+  specs:
+    classic_pagination (1.0.2)
```

Comment:
This is still showing `1.0.2`. Is this okay when gemspec is pointing at `1.0.4`?